catalog/transformer/catalog-transformer-pdf/pom.xml

```diff
@@ -25,7 +25,7 @@
     <packaging>bundle</packaging>
 
     <properties>
-        <pdfbox.version>2.0.2</pdfbox.version>
```

Comment:
❓ should we have this property defined in one place, like the root pom.xml?

Reply:
That's what I thought but I wasn't sure why we didn't do that in the first place. I'll try putting it in the root pom and see what happens.